docs/FindSphinx.cmake

```diff
@@ -0,0 +1,27 @@
+# CMake find_package() Module for Sphinx documentation generator
```

Comment:
where is this file used? I couldn't find a `find_package(Sphinx REQUIRED)` in this repository.

Reply:
This file helps to find SPHINX module

Reply:
some how find_package() function does not work on Windows

Reply:
but where is Sphinx even invoked by CMake? I thought it is purely used by `docs/Makefile`/Python. `FindSphinx` would be useful when we would do something like https://www.vortech.nl/en/integrating-sphinx-in-cmake/ where Sphinx is invoked by CMake. How are you currently generating the documentation? Maybe you missed some changes to `CMakeLists.txt`?